### DISCUSSION TITLE
lsp format

### DISCUSSION THREAD

**Discussion by Decodetalkers** [slint-ui/slint#2473] [Ideas]
Since there is a format tool, if can lsp format be applied to lsp?

**Comment by ogoffart** [comment]
There is a format tool in progress  in https://github.com/slint-ui/slint/tree/master/tools/fmt
We should include it in the LSP.

**Reply by Decodetalkers** [reply]
> There is a format tool in progress in https://github.com/slint-ui/slint/tree/master/tools/fmt We should include it in the LSP.

https://github.com/slint-ui/slint/pull/2475 I make a pull request, I make format to a crate, it can be used both in format tool and lsp
